eppo_client/base_model.py

```diff
@@ -1,14 +1,6 @@
-from pydantic import BaseModel
-
-
-def to_camel(s: str):
-    words = s.split("_")
-    if len(words) > 1:
-        return words[0] + "".join([w.capitalize() for w in words[1:]])
-    return words[0]
```

Comment:
glad they have an official version publish! our `to_camel` had a bug in it preventing test cases from passing